### PR TITLE
fix(condo): DOMA-3289 added spaces trimming for "number" and "account Number"… 

### PR DESCRIPTION
**Pull request by Alllex202**
… fields in meter modal form

Problem: you can save field "number" and "accountNumber" for "Meter" only with spaces or without spaces trimming at start and end

Also added this validation for import and fixed translates

### CODE REVIEW

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/meter/hooks/useUpdateMeterModal.tsx [hunk 1]
```diff
@@ -45,10 +45,16 @@ export const useUpdateMeterModal = (refetch) => {
     }, [selectedMeter])
 
     const handleSubmit = useCallback(values => {
+        const number = String(get(values, 'number', '')).trim()
+        const accountNumber = String(get(values, 'accountNumber', '')).trim()
```
Comment (AntonAL) [inline apps/condo/domains/meter/hooks/useUpdateMeterModal.tsx:49]: Why not to implement this trimming in `resolveInput` hook of `Meter` schema?

Comment (Alllex202) [reply apps/condo/domains/meter/hooks/useUpdateMeterModal.tsx:49]: Yes, it happens in `resolveInput` hook

File: apps/condo/domains/meter/hooks/useCreateMeterModal.tsx [hunk 2]
```diff
@@ -17,8 +18,13 @@ export function useCreateMeterModal (organizationId: string, propertyId: string,
 
     const handleMeterCreate = useCallback(values => {
         const numberOfTariffs = values.numberOfTariffs || 1
+        const number = String(get(values, 'number', '')).trim()
+        const accountNumber = String(get(values, 'accountNumber', '')).trim()
```
Comment (SavelevMatthew) [inline apps/condo/domains/meter/hooks/useCreateMeterModal.tsx:22]: Why explicit cast to string?

File: apps/condo/domains/meter/hooks/useUpdateMeterModal.tsx (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/meter/hooks/useUpdateMeterModal.tsx]: removed it from submit handler

File: apps/condo/domains/meter/hooks/useCreateMeterModal.tsx (hunk 2 above)
Comment (Alllex202) [reply apps/condo/domains/meter/hooks/useCreateMeterModal.tsx]: you're right, it's not needed here. Removed it

**Pull request merged**